sdk/lib/udmihelp/udmihelp.h

```diff
@@ -5,7 +5,14 @@
  * COPYRIGHT:   Copyright 2018 Stanislav Motylkov
  */
 
-#pragma once
+#ifndef UDMIHELP_H
+#define UDMIHELP_H
+
+#ifdef __cplusplus
+extern "C" {
```

Comment:
If you do this in udmihelp, it makes sense to also do that in dmilib.h (In case you want to use dmilib from c++ without udmihelp)

Reply:
Yep, I understand that, however dmilib is not used separately in C++ programs yet.